### PR TITLE
extra tests for iktomi/forms

### PR DESCRIPTION
**Pull request by boltnev**

### CODE REVIEW

File: iktomi/forms/widgets.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Widget(object):
     renders_hint = False
 
     def __init__(self, field=None, **kwargs):
-        if self._obsolete & set(kwargs):
+        if self._obsolete & set(kwargs):# pragma: no cover
```
Comment (ods) [inline iktomi/forms/widgets.py]: Why not test it with `multiple`?

Comment (boltnev) [inline iktomi/forms/widgets.py]: Should we test outdated parameters?

File: tests/forms/perms.py [hunk 2]
```diff
@@ -26,3 +26,26 @@ class F(Form):
...
+        class F(Form):
+            fields=[
+                Field('first', permissions='rw'),
+                Field('second'),
+            ]
+        env = AppEnvironment.create()
+        form = F(env, permissions='rwx')
+        first = form.get_field('first')
+        second = form.get_field('second')
+        self.assertEqual(first.perm_getter.get_perms(first), set(['r', 'w']))
```
Comment (ods) [inline tests/forms/perms.py]: `set('rw')` is more readable to me

**Comment by Harut** [comment]
Some tests fail under python3.5. Please fetch latest code and make sure all tests pass on both 2.7 and 3.5.

File: iktomi/forms/widgets.py (hunk 1 above)
Comment (Harut) [inline iktomi/forms/widgets.py]: this functionality is tested at least in fields. See `test_obsolete`.

File: tests/forms/widgets.py [hunk 3]
```diff
@@ -49,6 +49,14 @@ def test_init(self):
         for key, value in kwargs.items():
             self.assertEqual(value, getattr(widget, key))
 
+    def test_obsolete(self):
+        kwargs = dict(template='checkbox', multiple=True)
+        with self.assertRaises(TypeError) as exc:
+            widgets.Widget(**kwargs)
```
Comment (ods) [inline tests/forms/widgets.py]: What `kwargs` is constracted for? Why not just `widgets.Widget(template='checkbox', multiple=True)`?

**Pull request merged**